src/tests/system/tests/test_sss_ssh_knownhosts.py

```diff
@@ -0,0 +1,122 @@
+"""
+sss_ssh_knownhosts tests.
+
+:requirement: None
```

Comment:
There should a a requirement!

Reply:
That's why every PR touching tests should get QE review.

This PR was already closed. Please open a new PR with a proper fix.